tests/bsim/bluetooth/ll/cis/prj.conf

```diff
@@ -39,6 +38,7 @@ CONFIG_BT_CTLR_ISOAL_SINKS=9
 
 CONFIG_BT_CTLR_ISO_TX_BUFFERS=18
 CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=255
+CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=247
```

Comment:
```suggestion
CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=4095
```

Reply:
above two change was to see if it can fix #83586, but unfortunately no. I will remove the tests prj.conf changes, they are not required for this PR.